tests/Component/Transport/ConnectorTest.php

```diff
@@ -94,13 +94,11 @@ public function testSendError()
         );
         $this->mock->append($response);
 
-        $this->setExpectedException(
-            'Klarna\Rest\Transport\Exception\ConnectorException',
-            'ERR_1: msg1, msg2 (#cid_1)'
-        );
-
         $request = $this->connector->createRequest('http://somewhere/path', 'POST');
```

Comment:
Use constant from Method class instead of string

Reply:
fixed